firmware/application/apps/ui_flash_utility.cpp

```diff
@@ -126,21 +125,19 @@ std::filesystem::path FlashUtilityView::extract_tar(std::filesystem::path::strin
     return res;
 }
 
-void FlashUtilityView::flash_firmware(std::filesystem::path::string_type path) {
+bool FlashUtilityView::flash_firmware(std::filesystem::path::string_type path) {
```

Comment:
FYI, submitted PR 1827 to eliminate compiler warning.

Reply:
I didn't noticed the warning, sorry, and thanks for the fix.